.github/workflows/ci.yml

```diff
@@ -35,7 +36,7 @@ jobs:
       run: pip3 install codespell
 
     - name: Lint
-      run: cmake -D FORMAT_COMMAND=clang-format-12 -P cmake/lint.cmake
+      run: cmake -D FORMAT_COMMAND=clang-format-15 -P cmake/lint.cmake
```

Comment:
I would drop the specific version here and just use `clang-format`. Our Docker image determines the version anyway. This also frees us from having to update the version here as well if we change it in the image.

Reply:
Fixed by https://github.com/SpaceTeam/STS1_COBC_SW/pull/101/commits/444f3db7f08b2be3f1d66f0a98e2c57c9b7086b2.